### PR TITLE
taxonomy: Ingredient analysis translations

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: Ingredient analysis translations

### CODE REVIEW

**Comment by teolemon** [comment]
ERROR - ro:vegetarian already is associated to en:vegan (ingredients_analysis) - ro:vegetarian cannot be mapped to entry en:vegetarian

File: taxonomies/ingredients_analysis.txt [hunk 1]
```diff
@@ -91,32 +135,50 @@ ko:비건 채식
 nl:Veganistisch
 pl:Wegańskie
 pt:Vegano
-#ro:Vegetarian
+ro:Vegetarian
```
Comment (teolemon) [inline taxonomies/ingredients_analysis.txt:138]:
```suggestion
ro:Vegan
```

File: taxonomies/ingredients_analysis.txt [hunk 2]
```diff
@@ -182,14 +282,19 @@ hu:Vegetáriánus
...
-ru:Вегетарианское
 ro:Vegetarian
+ru:Вегетарианское
+sk:Vegetariánske
+sl:Vegetarijansko
+sr:Вегетаријанско
 sv:Vegetariskt
 tr:Vejetaryen
-th:มังสวิรัติ
+uk:Вегетаріанське
```
Comment (benbenben2) [inline taxonomies/ingredients_analysis.txt:297]: mistake?
```suggestion
th:มังสวิรัติ
uk:Вегетаріанське
```

File: taxonomies/ingredients_analysis.txt [hunk 3]
```diff
@@ -128,44 +190,82 @@ ar:غير النباتية
...
 es:Se desconoce si es vegano
+fi:Vegaaninen tila tuntematon
 fi:Vegaanisuus ei ole tiedossa
 fr:Caractère végétalien inconnu
 he:מצב טבעוני לא ידוע
 hu:Vegán státusz ismeretlen
+hr:Veganski status nepoznat
+hu:Vegán státusz ismeretlen
+it:Stato vegano sconosciuto
 it:Stato vegano sconosciuto
```
Comment (benbenben2) [inline taxonomies/ingredients_analysis.txt:242]:
```suggestion
hr:Veganski status nepoznat
hu:Vegán státusz ismeretlen
it:Stato vegano sconosciuto
```

File: taxonomies/ingredients_analysis.txt [hunk 4]
```diff
@@ -128,44 +190,82 @@ ar:غير النباتية
...
+ko:비건 상태 알 수 없음
+lt:Vegano statusas nežinomas
+lv:Vegāna statuss nav zināms
+lt:Vegano statusas nežinomas
+lv:Vegāna statuss nav zināms
+nl:Veganistische status onbekend
 nl:Veganistische status onbekend
+pl:Status wegański nieznany
 pt:Desconhece-se se é vegano
+pt:Desconhece-se se é vegano
```
Comment (benbenben2) [inline taxonomies/ingredients_analysis.txt:254]:
```suggestion
ja:ビーガンか不明
ko:비건 상태 알 수 없음
lt:Vegano statusas nežinomas
lv:Vegāna statuss nav zināms
nl:Veganistische status onbekend
pl:Status wegański nieznany
pt:Desconhece-se se é vegano
```

**Pull request merged**